src/biome/text/modules/heads/token_classification.py

```diff
@@ -158,30 +163,14 @@ def featurize(
 
         return instance
 
-    @staticmethod
-    def _bioul2bio(labels: List[str]):
-        return [tag.replace("L-", "I-", 1).replace("U-", "B-", 1) for tag in labels]
-
     def task_name(self) -> TaskName:
         return TaskName.token_classification
 
-    def decode(self, output: TaskOutput) -> TaskOutput:
-        output.tags = (
-            [
-                [
-                    vocabulary.label_for_index(self.backbone.vocab, idx)
-                    for idx in tags[0]
-                ]
-                for instance_k_tags in output.k_tags
-                for tags in instance_k_tags
-            ],
-        )
-
-        del output.k_tags
-        return output
-
     def forward(  # type: ignore
-        self, text: TextFieldTensors, labels: torch.IntTensor = None
+        self,
+        text: TextFieldTensors,
+        raw_text: Union[str, List[str]],
```

Comment:
If I undestand, `raw_text` contains texts/tokens related to input batch, so `Union[List[str], List[List[str]]]` ?